databuilder/models/table_column_usage.py

```diff
@@ -64,7 +61,7 @@ def __init__(self,
                 raise NotImplementedError('Column is not supported yet {}'.format(col_readers))
 
         self.col_readers = col_readers
-        self._node_iterator = self._create_next_node()
+        self._node_iterator = iter(self.create_nodes())
```

Comment:
Did you need to add `iter` here? I thought yield should generate iterator by itself.